app/models/concerns/course/lesson_plan/item_todo_concern.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module Course::LessonPlan::ItemTodoConcern
+  extend ActiveSupport::Concern
+
+  included do
+    after_save do |item|
+      if item.try(:actable_type).try(:constantize).try(:has_todo?) && item.draft_changed?
```

Comment:
Actually I used `constantize` partly cause I think the `has_todo?` is more of a property of the class (ie. `Course::Assessment`), rather than an instance of the class. Come to think of it, an instance method is also okay, but it requires the instantiation of an object to check.  

I'm not sure if there are performance benefits/issues because most of the time the actable object would have been loaded to trigger the update on `lesson_plan_item`, which then checks on `has_todo?`.

If we want to do `item.has_todo?`, we will have to `delegate`, because this `has_todo?` method should be implemented on the `actable_object` and not `lesson_plan_item`.